тайловые форматы экспорта

tarwirdur opened the pull request:
Просьба сделать code review.
Текущее состояние выложено http://nkrt.mikhe.ru:22837

wladich commented:
Пожалуйста, в первой строке commit message сделайте краткое описание. Т.е. примерно:
Added ability to export to RMaps format #37

wladich commented:
В прошлой версии выбор формата был сделан в виде select. Я думаю, так и надо было оставить.
Пока форматов всего два, radio button, конечно, удобнее, но я надеюсь, что форматов станет больше и тогда всё равно придётся делать select

wladich commented:
Код внимательно пока не смотрел

tarwirdur commented on src/lib/leaflet.control.export/control.css:
```diff
@@ -0,0 +1,117 @@
+    border-top: 1px solid #eee;
+}
+
+/* Styles from leaflet.control.printPages */
```
Хочется избежать дублирования стилей, но не понятно, куда их можно вынести.
Кстати, нет планов использовать какой-нибудь scss?

tarwirdur commented:
> Added ability to export to RMaps format #37

fixed

> В прошлой версии выбор формата был сделан в виде select. Я думаю, так и надо было оставить.

fixed

> http://nkrt.mikhe.ru:22837

updated

> Про надпись на кнопке: пока форматов всего два, думаю, можно сделать JNX/RMap в две строки

В две строки красиво не получилось, в одну, на мой взгляд, нормально выглядит. Хотелось бы иконку хорошую найти.

tarwirdur commented:
Всё ещё жду замечаний к коду. После вместе с имправлениями, могу сделать #41, добавить заодно и экспорт в mbtiles (для ios вроде есть [приложения](https://itunes.apple.com/ru/app/mbtiles-gps/id592703465?mt=8) которые его едят).

Тогда можно будет закрыть сразу #37 #41 #77

wladich commented:
Постараюсь на выходных заняться сайтом...

tarwirdur commented:
В качестве полезной нагрузке к пингу добавляю сюда коммит к #41. Внешне можно посмотреть по ссылке из первого сообщения.

tarwirdur commented:
rebased to master

myadzel commented:
Есть какие-то планы по внедрению экспорта в sqlitedb?

wladich commented:
Я смотрел на PR, мне в нём что-то не понравилось, я решил его переделать, используя куски от имеющегося, но завяз...
А так планы есть, да...

tarwirdur commented:
Если оставишь комментарии - попробую поправить.

wladich commented:
Сейчас всё не помню, но главная проблема была, что всё это пришло одним здоровым коммитом который очень сложно осознать. При этом часть изменений это просто перестановка и типовые замены, но поскольку всё это в одном коммите с новым кодом, то проверить очень тяжело

wladich commented:
В #259 я начал делить изменения по коммитам, но не доделал

tarwirdur commented:
Отдельные коммиты можно увидеть в старом pull request: https://github.com/wladich/nakarte/pull/142/commits

tarwirdur commented:
meh...

> В #259 я начал делить изменения по коммитам, но не доделал

Я могу как-то помочь?

tarwirdur commented:
Закрываю всвязи с новым PR: https://github.com/wladich/nakarte/pull/640
